x/clock/abci.go

```diff
@@ -19,9 +19,11 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
 
 	p := k.GetParams(ctx)
 
-	errorExecs := make([]string, len(p.ContractAddresses))
+	contracts := k.GetAllContracts(ctx, false)
+	errorExecs := make([]string, len(contracts))
```

Comment:
See line 23. If more than one contract exists, the length of errorExecs will always be greater than 0.